src/main/kotlin/com/example/Employee/services/BankAccountService.kt

```diff
@@ -30,9 +30,9 @@ public class BankAccountService(private val employeeRepository: EmployeeReposito
                 employee = employee.get()
             )
         )
-
+        return bankAccountRepository.findById(savedAccount.id).get()
```

Comment:
Although this is technically right, is there any reason to read the bankAccount from the database again, when the entity returned from the `.save()` method also serves the same response we need?

Reply:
I was just checking something. Changed it back.